setup.py

```diff
@@ -23,7 +23,7 @@
 
 setup(
     name='infoblox-client',
-    version='0.4.22',
+    version='0.4.23',
```

Comment:
Feel like this would be `0.5.0`?

Reply:
yes, even i feel this should be 0.5.0 as it is a new feature addition